staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -448,6 +473,65 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	// Required.
 	// +listType=set
 	ValidationActions []ValidationAction `json:"validationActions,omitempty" protobuf:"bytes,4,rep,name=validationActions"`
+
+	// PerNamespaceParamRef specifies the parameter resource or resources used to
+	// configure the admission control policy. It should point to a resources of
+	// the type specified in ParamKind of the bound ValidatingAdmissionPolicy.
+	//
+	// If the policy specifies a ParamKind and the resource referred to by
+	// PerNamespaceParamRef does not exist, whether this binding fails the request
+	// depends upon the ParameterNotFoundAction of the perNamespaceParamRef
+	//
+	// If the paramKind of the policy referred to by `policyName` is cluster scoped,
+	// and perNamespaceParamRef set, the binding is considered mis-configured, and
+	// the failureMode applies.
+	//
+	// `perNamespaceParamRef` and `paramRef` are members of a union; if one of the
+	// fields is set, the other must be unset.
+	// +optional
+	PerNamespaceParamRef *PerNamespaceParamRef `json:"perNamespaceParamRef,omitempty" protobuf:"bytes,5,opt,name=perNamespaceParamRef"`
+}
```

Comment:
It didn't align with what states in pkg/apis/admissionregistration/types.go

Reply:
removed this field